assesspy/metrics.py

```diff
@@ -132,7 +132,9 @@ def _calculate_prb(
         lambda x: math.log2(x / 2)
     )
 
-    prb_model = sm.OLS(lhs.to_numpy(), rhs.to_numpy()).fit()
+    prb_model = sm.OLS(
+        endog=lhs.to_numpy(), exog=sm.tools.tools.add_constant(rhs.to_numpy())
+    ).fit(method="qr")
```

Comment:
This was the crux of the PRB model issue. statsmodel doesn't add an intercept by default (unlike lm()). The method change to "qr" was just to match R, but doesn't seem to actually make a difference.

Reply:
**[Praise]** Nice catch! This is a super annoying interface.